src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/_advanced_parameters.yml

```
@@ -45,3 +45,7 @@ _employee:
 _profiles:
     resource: "profiles.yml"
     prefix: /profiles
+
+_experimental_feature:
+    resource: "experimental_feature.yml"
+    prefix: /experimental_feature
```

```suggestion
    prefix: /experimental-feature
```
It's two words, not one :)